lib/classes/CLI.js

```diff
@@ -205,6 +211,18 @@ class CLI {
     this.consoleLog(chalk.dim('* Pass "--verbose" to this command to get in-depth plugin info'));
     this.consoleLog(chalk.dim('* Pass "--no-color" to disable CLI colors'));
     this.consoleLog(chalk.dim('* Pass "--help" after any <command> for contextual help'));
+    this.consoleLog('');
+
+    this.consoleLog(chalk.yellow.underline('Interactive Quickstart'));
+    this.consoleLog(
+      chalk.dim(
+        `* Run serverless (or shortcut sls) without any arguments to initialize an interactive setup
+  of functionalities related to given service or current environment`
+      )
+    );
+    this.consoleLog(
+      chalk.dim('* Pass "--help-interactive" for contextual help on interactive CLI options')
```

Comment:
I've checked `sls --help-interactive` and received same output as `sls --help`.

As I see it's due to invalid option check at L121

Reply:
Hmm. Weird. works fine afaict 
![image](https://user-images.githubusercontent.com/667763/67282866-9c8de380-f4a0-11e9-867d-6dd239f762b8.png)

Reply:
ohh.. 🤦‍♂